packages/quantification/src/__tests__/quantification.test.ts

```diff
@@ -494,3 +495,216 @@ describe('getQuantificationSummary', () => {
     });
   });
 });
+
+describe('getQuantificationSummaries', () => {
+  it('will get the tonnes that are grandfatherable given a COMET output file for 5 grandfatherable years', async () => {
+    let modelRun = GRANDFATHERABLE_YEARS_OUTPUT.Day.Cropland.ModelRun;
+    modelRun = Array.isArray(modelRun) ? modelRun[0] : modelRun;
```

Comment:
Why can't this be a `const`? Don't we know the structure of the GRANDFATHERABLE_YEARS_OUTPUT fixture?

Reply:
The fixture, yes, but the type is one of those `ModelRun | ModelRun[]` type things

Reply:
Ah